pkg/scheduler/apis/config/scheme/scheme_test.go

```diff
@@ -42,347 +41,6 @@ func TestCodecsDecodePluginConfig(t *testing.T) {
 		wantErr      string
 		wantProfiles []config.KubeSchedulerProfile
 	}{
-		// v1beta2 tests
-		{
-			name: "v1beta2 all plugin args in default profile",
-			data: []byte(`
```

Comment:
We have all the similar testcases in v1beta3, right?

Reply:
Yes, we have.